miio/device.py

```diff
@@ -333,7 +139,7 @@ def raw_command(self, command, parameters):
 
         :param str command: Command to send
         :param dict parameters: Parameters to send"""
-        return self.send(command, parameters)
+        return self.protocol.send(command, parameters)
```

Comment:
I'm starting to have second thoughts on deprecating the `send`. This code would generally be much cleaner if we'd keep it as it is, and simply make it wrap the call to the private protocol. Thoughts on this? What are the pros for deprecating it (and making all other methods more complicated)?

Reply:
I guess the main reason why I decided to do that was to do less inheritance and more composition (to make things easier to follow). Keeping `send`, however, makes it more backwards compatible.

I've removed the deprecation.